devise-security.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.3.0'
 
   s.add_runtime_dependency 'devise', '>= 4.3.0', '< 5.0'
+  s.add_runtime_dependency 'rails', '>= 4.2.0', '< 7.0'
```

Comment:
@olbrich I'd like to remove this and add it in a separate PR. TODO is captured by https://github.com/devise-security/devise-security/issues/311.